lib/runners/schema/options.rb

```diff
@@ -0,0 +1,18 @@
+module Runners
+  module Schema
+    Options = _ = StrongJSON.new do
+      # @type self: Types::Options
+
+      let :source, enum?(
```

Comment:
[question] `source` is required?

```suggestion
      let :source, enum(
```